blank_react_project/package.json

```diff
@@ -26,7 +26,8 @@
     "deploy:pages": "gh-pages -d build",
     "deploy": "npm run deploy:contract && npm run deploy:pages",
     "build": "react-scripts build && gulp",
-    "dev:deploy:contract": "near dev-deploy ./out/main.wasm"
+    "dev:deploy:contract": "near dev-deploy ./out/main.wasm",
+    "postinstall": "mv near.gitignore .gitignore"
```

Comment:
Not sure how this should work. Like this is a script for template project, my understanding is that it won't be executed during project generation. It would be executed when someone installs generated project though. But we want `.gitignore` at time of project generation.